.github/workflows/ReleaseStaticBinaries.yml

```diff
@@ -0,0 +1,47 @@
+name: Release static binaries
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: write
+
+jobs:
+  release-binaries:
+    name: Release ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-musl
+          - os: macos-latest
+            target: x86_64-apple-darwin
+          - os: macos-latest
+            target: aarch64-apple-darwin
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Rust
+      uses: dtolnay/rust-toolchain@master
+      with:
+        toolchain: stable
+        target: ${{ matrix.target }}
+
+    - name: Install cargo-rs
+      shell: bash
+      run: |
+        cargo install cross
+
+    - name: Build
+      run: cross build --target ${{ matrix.target }} --release
+
+    - name: Upload binaries to GitHub Release
+      uses: svenstaro/upload-release-action@v2
+      with:
+        repo_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
By the way: for this repo, I am using `secrets.GH_PAT` instead of `secrets.GITHUB_TOKEN`.

Other than that it looks very good. Will merge after the changes.

Reply:
GitHub automatically injects the `GITHUB_TOKEN` secret. At my fork I didn't need to do any configuration. 

> Before each job begins, GitHub fetches an installation access token for the job. The GITHUB_TOKEN expires when a job finishes or after a maximum of 24 hours.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
Ahh.. That's interesting, didn't know that, thanks for the explanation.